starfish/intensity_table.py

```diff
@@ -276,3 +278,58 @@ def synthetic_intensities(
         intensities[cls.Constants.GENE.value] = ('features', genes)
 
         return intensities
+
+    @classmethod
+    def from_image_stack(cls, image_stack, crop_x: int=0, crop_y: int=0, crop_z: int=0) -> "IntensityTable":
+        """Generate an IntensityTable from all the pixels in the ImageStack
+
+        Parameters
+        ----------
+        crop_x : int
+            number of pixels to crop from x
```

Comment:
This is potentially confusing because one could interpret this as "I will crop crop_x/2 pixels from the left and crop_x/2 pixels from the right, for a total of crop_x pixels."